### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by mariaruccolo** [MorphoSource/Community#123] [Q&A] [answered]
How do I extract a scale from the Morpho Source Images?

**Comment by JulieWinchester** [comment]
The answer probably depends on what kinds of images you're trying to parse scale from, and maybe on what the specific media contributors have provided with their media. Can you link examples of specific media you're interested in? If so, I can try to give a more specific answer.

**Comment by mariaruccolo** [comment]
I am trying to add a scale to the image for Media 000433745: Cranium. The link is https://www.morphosource.org/concern/media/000433745?locale=en. Thank you!

**Comment by JulieWinchester** [comment, marked as answer]
<img width="499" alt="Screenshot 2024-09-13 at 2 20 25 PM" src="https://github.com/user-attachments/assets/cf9cc426-b82c-4422-9572-8a092e562800">

The scale for this CT stack is found in the File Object Details section, specifically the X/Y/Z Pixel Spacing fields. These fields define how "wide" each pixel is (in millimeters, as indicated by the Pixel Spacing Units field) in the X, Y, and Z dimensions. So each pixel in this CT stack has a length of 0.071155 millimeters in each dimension (X, Y, and Z).

**Reply by mariaruccolo** [reply]
Thank you, that is very helpful. Is there a way that I can make a scale bar appear on the Media Display?

**Reply by JulieWinchester** [reply]
<img width="572" alt="Screenshot 2024-09-13 at 2 49 26 PM" src="https://github.com/user-attachments/assets/87b803c4-5668-4b6b-abd5-1a951abd68d0">

Its not exactly a scale bar, but you can use the Nodes menu of the viewer to place one landmark node, then hold shift and click a second time to create a second node linked to the first. Hovering over the connection linking the two nodes will show the length between the nodes, which will provide scale information. For CT scans, it's best to place the nodes while scrolling through slices and then load into volume view, since placing nodes in volume view is harder.

**Reply by mariaruccolo** [reply]
That is exactly what I needed thank you again!!
